README.md

```diff
@@ -1,7 +1,11 @@
-# Event Buddy
+# ![alt Event Buddy](/src/assets/logo.png)
```

Comment:
@udit-001 These commits have already been merged to the master. Please clear your history. While pulling from upstream use `git pull --rebase upstream master`

Reply:
I don't understand how to proceed further, perhaps a more detailed instruction would help. ;/

Reply:
Run `git rebase -i HEAD~2` command and delete the commit that is already merged & then pull from upstream using `git pull --rebase upstream`

Reply:
@geekyd It seems good to go now!